man/systemd.special.xml

```diff
@@ -548,7 +557,8 @@
             <para>Similar to <filename>cryptsetup.target</filename>, but for encrypted
             devices which are accessed over the network. It is used for
             <citerefentry><refentrytitle>crypttab</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-            entries marked with <option>_netdev</option>.</para>
+            entries marked with <option>_netdev</option>.
+            See <filename>initrd-cryptsetup.target</filename> for the initramfs equivalent.</para>
```

Comment:
in the systemd documentation we usually say "initrd" and not "initramfs". both terms are slightly misleading, since the backing fs is tmpfs these days and neither a ramdisk nor a ramfs. But we decided to just use "initrd" everywhere as the original term and slightly less misleading since "rd" can mean so much more...

Reply:
Sure, WFM. I found another instance of "initramfs" which I switched to "initrd".